### ISSUE TITLE
docker-compose build fails on fresh install

### ISSUE THREAD

**Issue by rob-luke** [mne-tools/mne-docker#3]
Hi,

I just cloned this repository to my computer and installed docker and ran the line 'docker-compose build' and it threw the following error. I am using a Mac if that's relevant.

I am keen to set up a docker image that can run MNE-Python, so Im available to help debug this issue, but I am new to docker, so any direction is appreciated.

```text
git:(main) docker-compose build
Building scheduler
[+] Building 171.6s (12/12) FINISHED
 => [internal] load build definition from Dockerfile                                                                    0.0s
 => => transferring dockerfile: 813B                                                                                    0.0s
 => [internal] load .dockerignore                                                                                       0.0s
 => => transferring context: 2B                                                                                         0.0s
 => [internal] load metadata for docker.io/continuumio/miniconda3:4.8.2                                                 9.7s
 => [1/7] FROM docker.io/continuumio/miniconda3:4.8.2@sha256:456e3196bf3ffb13fee7c9216db4b18b5e6f4d37090b31df3e030992  30.7s
 => => resolve docker.io/continuumio/miniconda3:4.8.2@sha256:456e3196bf3ffb13fee7c9216db4b18b5e6f4d37090b31df3e0309926  0.0s
 => => sha256:456e3196bf3ffb13fee7c9216db4b18b5e6f4d37090b31df3e0309926e98cfe2 953B / 953B                              0.0s
 => => sha256:b4adc22212f1c4628a1922a5669640e86734d0da3c26a0c35cb034e388c033ae 3.19kB / 3.19kB                          0.0s
 => => sha256:68ced04f60ab5c7a5f1d0b0b4e7572c5a4c8cce44866513d30d9df1a15277d6b 27.09MB / 27.09MB                       15.3s
 => => sha256:9c388eb6d33c40662539172f0d9a357287bd1cd171692ca5c08db2886bc527c3 80.22MB / 80.22MB                       25.1s
 => => sha256:96cf53b3a9dd496f4c91ab86eeadca2c8a31210c2e5c2a82badbb0dcf8c8f76b 50.36MB / 50.36MB                       20.6s
 => => extracting sha256:68ced04f60ab5c7a5f1d0b0b4e7572c5a4c8cce44866513d30d9df1a15277d6b                               0.9s
 => => extracting sha256:9c388eb6d33c40662539172f0d9a357287bd1cd171692ca5c08db2886bc527c3                               3.3s
 => => extracting sha256:96cf53b3a9dd496f4c91ab86eeadca2c8a31210c2e5c2a82badbb0dcf8c8f76b                               1.9s
 => [internal] load build context                                                                                       0.0s
 => => transferring context: 871B                                                                                       0.0s
 => [2/7] RUN conda install --yes     -c conda-forge     python==3.8     python-blosc     cytoolz     dask==2021.4.0  108.1s
 => [3/7] RUN pip install s3fs                                                                                         10.1s
 => [4/7] RUN pip install mne                                                                                           9.4s
 => [5/7] RUN pip install bokeh                                                                                         0.6s
 => [6/7] COPY prepare.sh /usr/bin/prepare.sh                                                                           0.0s
 => [7/7] RUN mkdir /opt/app                                                                                            1.2s
 => exporting to image                                                                                                  1.7s
 => => exporting layers                                                                                                 1.7s
 => => writing image sha256:305f452ab8147df10a30a91fa8d2a687fd912af928604533c6edb2055c7d7286                            0.0s
 => => naming to docker.io/mnetools/mne-python                                                                          0.0s

Use 'docker scan' to run Snyk tests against images to find vulnerabilities and learn how to fix them
Building worker
[+] Building 1.1s (12/12) FINISHED
 => [internal] load build definition from Dockerfile                                                                    0.0s
 => => transferring dockerfile: 37B                                                                                     0.0s
 => [internal] load .dockerignore                                                                                       0.0s
 => => transferring context: 2B                                                                                         0.0s
 => [internal] load metadata for docker.io/continuumio/miniconda3:4.8.2                                                 1.0s
 => [1/7] FROM docker.io/continuumio/miniconda3:4.8.2@sha256:456e3196bf3ffb13fee7c9216db4b18b5e6f4d37090b31df3e0309926  0.0s
 => [internal] load build context                                                                                       0.0s
 => => transferring context: 32B                                                                                        0.0s
 => CACHED [2/7] RUN conda install --yes     -c conda-forge     python==3.8     python-blosc     cytoolz     dask==202  0.0s
 => CACHED [3/7] RUN pip install s3fs                                                                                   0.0s
 => CACHED [4/7] RUN pip install mne                                                                                    0.0s
 => CACHED [5/7] RUN pip install bokeh                                                                                  0.0s
 => CACHED [6/7] COPY prepare.sh /usr/bin/prepare.sh                                                                    0.0s
 => CACHED [7/7] RUN mkdir /opt/app                                                                                     0.0s
 => exporting to image                                                                                                  0.0s
 => => exporting layers                                                                                                 0.0s
 => => writing image sha256:305f452ab8147df10a30a91fa8d2a687fd912af928604533c6edb2055c7d7286                            0.0s
 => => naming to docker.io/mnetools/mne-python                                                                          0.0s

Use 'docker scan' to run Snyk tests against images to find vulnerabilities and learn how to fix them
Building base-notebook
[+] Building 0.0s (1/1) FINISHED
 => ERROR [internal] load git source github.com/jupyter/docker-stacks.git#master:base-notebook                          0.0s
------
 > [internal] load git source github.com/jupyter/docker-stacks.git#master:base-notebook:
------
failed to solve with frontend dockerfile.v0: failed to read dockerfile: failed to load cache key: subdir not supported yet
ERROR: Service 'base-notebook' failed to build : Build failed
```

**Comment by rob-luke** [comment]
The notebook causing the build to fail can now be viewed in the CI build at https://github.com/mne-tools/mne-docker/runs/2836721778

**Comment by rob-luke** [comment]
Closed with #5
